.github/workflows/build.yml

```diff
@@ -311,7 +314,9 @@ jobs:
       - name: Run micro-benchmarks
         run: cargo bench --locked --all-features
         env:
-          RUSTFLAGS: --cfg=bench
+          # XXX TBD ??? ???
+          # RUSTFLAGS: --cfg=bench
+          RUSTFLAGS: --cfg bench --cfg unstable_use_arc_from_stdlib
```

Comment:
I think this would be better (would need to add `rust-flags` item to the matrix):

```diff
-          RUSTFLAGS: --cfg bench --cfg unstable_use_arc_from_stdlib
+          RUSTFLAGS: --cfg=bench ${{ matrix.rust-flags }}
```

Quicker workaround could be something like this:

```diff
-          RUSTFLAGS: --cfg bench --cfg unstable_use_arc_from_stdlib
+          RUSTFLAGS: --cfg=bench ${{ env.CFG_FLAGS }}
```

(would need to define env `CFG_FLAGS` somewhere to do this)